tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -53,23 +51,23 @@
     "headers": {
         "accept": "*/*",
         "connection": "close",
-        "content-length": "573",
+        "content-length": "552",
         "content-type": "application/x-www-form-urlencoded",
         "host": "api.va.gov",
-        "i-twilio-idempotency-token": "62c29575-5ed0-4638-92cb-297c5c8763ea",
+        "i-twilio-idempotency-token": "09f6d617-b893-4864-8f42-24a36ec48691",
         "user-agent": "TwilioProxy/1.1",
-        "x-amzn-trace-id": "Self=1-62cda3f5-20dbbea14d3b70cf717c9a77;Root=1-62cda3f5-44e716a47645d83a41a7ee7f",
-        "x-forwarded-for": "44.202.129.169, 10.237.28.71, 44.202.129.169, 10.247.32.239",
+        "x-amzn-trace-id": "",
+        "x-forwarded-for": "",
         "x-forwarded-host": "api.va.gov:443",
         "x-forwarded-port": "443",
         "x-forwarded-proto": "https",
         "x-forwarded-scheme": "https",
         "x-home-region": "us1",
-        "x-real-ip": "44.202.129.169",
-        "x-twilio-signature": "twiliosig",
+        "x-real-ip": "",
+        "x-twilio-signature": "CaQLzIRYCf7qUkE76KNq7c3kCSU=",
+        "x-use-static-proxy": "true"
     },
-    "body": "VG9Db3VudHJ5PVVTJlRvU3RhdGU9JlNtc01lc3NhZ2VTaWQ9U002NDI0Yjc5ZDY3NTlkYjU3MWI0OGUzMTZjYWVmNDlmMyZOdW1NZWRpYT0wJlRvQ2l0eT0mRnJvbVppcD05NjE1MCZTbXNTaWQ9U002NDI0Yjc5ZDY3NTlkYjU3MWI0OGUzMTZjYWVmNDlmMyZGcm9tU3RhdGU9TlYmU21zU3RhdHVzPXJlY2VpdmVkJkZyb21DaXR5PUNBUlNPTitDSVRZJkJvZHk9VkFYJkZyb21Db3VudHJ5PVVTJlRvPTgwNzI4Jk1lc3NhZ2luZ1NlcnZpY2VTaWQ9TUdmZWQ1NjhmMmQzZGM0YWM2ODUwYTBiYWQyYzk4NzNiMSZUb1ppcD0mQWRkT25zPSU3QiUyMnN0YXR1cyUyMiUzQSUyMnN1Y2Nlc3NmdWwlMjIlMkMlMjJtZXNzYWdlJTIyJTNBbnVsbCUyQyUyMmNvZGUlMjIlM0FudWxsJTJDJTIycmVzdWx0cyUyMiUzQSU3QiU3RCU3RCZOdW1TZWdtZW50cz0xJlJlZmVycmFsTnVtTWVkaWE9MCZNZXNzYWdlU2lkPVNNNjQyNGI3OWQ2NzU5ZGI1NzFiNDhlMzE2Y2FlZjQ5ZjMmQWNjb3VudFNpZD1BQzU1MWZjNjA4NmY5M2E4MzQ5MjQ2NmZjNjA0YzY4YWY0JkZyb209JTJCMTc3NTcyMDYwMjAmQXBpVmVyc2lvbj0yMDEwLTA0LTAx",
-    "isBase64Encoded": True,
+    "body": "VG9Db3VudHJ5PVVTJlRvU3RhdGU9JlNtc01lc3NhZ2VTaWQ9U005NWFmM2U3NzBkYTg5OGMyMGUwM2RmNDQ4NzAzMjgzYiZOdW1NZWRpYT0wJlRvQ2l0eT0mRnJvbVppcD02NTYyNSZTbXNTaWQ9U005NWFmM2U3NzBkYTg5OGMyMGUwM2RmNDQ4NzAzMjgzYiZGcm9tU3RhdGU9TU8mU21zU3RhdHVzPXJlY2VpdmVkJkZyb21DaXR5PUNBU1NWSUxMRSZCb2R5PVk2NSZGcm9tQ291bnRyeT1VUyZUbz01MzA3OSZNZXNzYWdpbmdTZXJ2aWNlU2lkPU1HYmU2YmIyN2E5Y2ExNWRlMjc2YzViODZmMGQ5ZTljMmUmVG9aaXA9JkFkZE9ucz0lN0IlMjJzdGF0dXMlMjIlM0ElMjJzdWNjZXNzZnVsJTIyJTJDJTIybWVzc2FnZSUyMiUzQW51bGwlMkMlMjJjb2RlJTIyJTNBbnVsbCUyQyUyMnJlc3VsdHMlMjIlM0ElN0IlN0QlN0QmTnVtU2VnbWVudHM9MSZNZXNzYWdlU2lkPVNNOTVhZjNlNzcwZGE4OThjMjBlMDNkZjQ0ODcwMzI4M2ImQWNjb3VudFNpZD1BQzU1MWZjNjA4NmY5M2E4MzQ5MjQ2NmZjNjA0YzY4YWY0JkZyb209JTJCMTQxNzg0NjY0MTgmQXBpVmVyc2lvbj0yMDEwLTA0LTAx"
```

Comment:
@edDocMe360 
Please DM me about this.

Reply:
{'AccountSid': 'twilioaccountsid', 'ApiVersion': '2010-04-01', 'From': '+2222222222', 'MessageSid': 'somemessageidentifier', 'MessageStatus': 'sent', 'SmsSid': 'thisissomesmsid', 'SmsStatus': 'sent', 'To': '+11111111111'}